test/__init__.py

```
@@ -1,2 +1,3 @@
 from .test_rbac import *
 from .test_oauth_url import *
+from .test_client_origins import *
```

Ruff (0.16.1)

[error] `from .test_client_origins import *` used; unable to detect undefined names

(F403)